docs/content/installation/migration/ingress-nginx.md

```diff
@@ -0,0 +1,549 @@
+```
+
+---
+
+**Ingress-NGINX Controller**
+```yaml
+nginx.ingress.kubernetes.io/proxy-ssl-secret: "secretName"
+nginx.ingress.kubernetes.io/proxy-ssl-verify: "on|off"
+nginx.ingress.kubernetes.io/proxy-ssl-verify-depth: "1"
+nginx.ingress.kubernetes.io/proxy-ssl-protocols: "TLSv1.2"
+nginx.ingress.kubernetes.io/proxy-ssl-ciphers: "DEFAULT"
+nginx.ingress.kubernetes.io/proxy-ssl-name: "server-name"
+nginx.ingress.kubernetes.io/proxy-ssl-server-name: "on|off"
+```
+
+**NGINX Ingress Controller**
+```yaml
+egressMTLS:
+   tlsSecret: secretName
+
+   verifyServer: true|false
+
+   verifyDepth: 1
+
+   protocols: TLSv1.2
+
+   ciphers: DEFAULT
+
+   sslName: server-name
+
+   serverName: true|false
+```
+
+---
+
+#### Session Persistence with NGINX Plus
+With [NGINX Plus]({{<relref "overview/nginx-plus">}}), you can use [Policy]({{<relref "configuration/policy-resource">}}) resources for session persistence, which have corresponding annotations for the community Ingress-NGINX Controller.
+
+**Ingress-NGINX Controller**
+```yaml
+nginx.ingress.kubernetes.io/affinity: "cookie"
+nginx.ingress.kubernetes.io/session-cookie-name: "cookieName"
+nginx.ingress.kubernetes.io/session-cookie-expires: "x"
+nginx.ingress.kubernetes.io/session-cookie-path: "/route"
+nginx.ingress.kubernetes.io/session-cookie-secure: "true"
+```
+
+**NGINX Ingress Controller**
+```yaml
+sessionCookie:
+  enable: true
+
+  name: cookieName
+
+  expires: xh
+
+  path: /route
+
+  secure: true
+```
+
+## Migration with Kubernetes Ingress Resources
+The other option for migrating from the community Ingress-NGINX Controller to NGINX Ingress Controller is using only [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) and [ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/) from standard Kubernetes resources, potentially relying on [mergeable Ingress types](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/ingress-resources/mergeable-ingress-types).
+
+This ensures that all configuration is kept in the Ingress object.
+
+{{< warning >}}
+Do not alter the `spec` field of the Ingress resource when taking this option.
+{{< /warning >}}
+
+### Advanced Configuration with Annotations
+This table maps the Ingress-NGINX Controller annotations to NGINX Ingress Controller's equivalent annotations, and the respective NGINX Directive.
+
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
+| Ingress-NGINX Controller | NGINX Ingress Controller | NGINX Directive |
+| ------------------------ | ------------------------ | --------------- |
+| [`nginx.ingress.kubernetes.io/configuration-snippet`](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#configuration-snippet) | [`nginx.org/location-snippets`](https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#snippets-and-custom-templates) | N/A |
```

Comment:
Do these need to be formatted as in-line code or perhaps bold or italic? I know we've gone back and forth on this. We should decide.

Reply:
I've formatted these all as in-line code to march the original blog post.

https://www.nginx.com/blog/migrating-from-community-ingress-controller-to-f5-nginx-ingress-controller/

Reply:
The blog post isn't a style guide. I raised the issue in Slack so we have consensus.

Reply:
The table on Proxying and Load Balancing doesn't use in-line code for parameters, making it inconsistent with other tables in the same document. This inconsistency is hard to miss.